server/src/main/resources/routes

```diff
@@ -3,6 +3,8 @@
 # ~~~~
 
 # Home page
+ +nocsrf
```

Comment:
What are these ` +nocsrf` lines?

Reply:
by default with play.filters.enabled with play.filters.csp.CSPFilter all the request POST and delete expect to have csrf token . It it is not represent it throw 430. Either we need add csp token or ignore csrf check for the request